framework/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ Yii Framework 2 Change Log
 2.0.11 under development
 ------------------------
 
+- Enh #12798: Changed `yii\cache\Dependency::getHasChanged()` (to be deprecated in 2.1) to `yii\cache\Dependency::isChanged()` (dynasource)
```

Comment:
"Deprecate `yii\cache\Dependency::getHasChanged()` in favor of `yii\cache\Dependency::isChanged()`"?